src/pages/reader/pdf.vue

```diff
@@ -75,28 +87,30 @@ export default {
         }
       }
     },
+    async getFileBuffer() {
+      let buffer;
+      if (window.caches) {
+        try {
+          const cache = await caches.open('reader-epub');
```

Comment:
PDF uses `reader-epub` too, how about a generic name for the cache? e.g. `reader-ebook`?

Reply:
actually it is suppose to be `reader-pdf`...